twister:
    #Publish all joint states
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 30

    #Position controllers
    right_front_wheel_to_chassis_controller:
        type: effort_controllers/JointPositionController
        joint: right_wheel_front_joint
        pid: {p: 50.0, i: 1, d: 10.0}

    left_front_wheel_to_chassis_controller:
        type: effort_controllers/JointPositionController
        joint: left_wheel_front_joint
        pid: {p: 50.0, i: 1, d: 10.0}

    right_back_wheel_to_chassis_controller:
        type: effort_controllers/JointPositionController
        joint: right_wheel_back_joint
        pid: {p: 50.0, i: 1, d: 10.0}

    left_back_wheel_to_chassis_controller:
        type: effort_controllers/JointPositionController
        joint: left_wheel_back_joint
        pid: {p: 50.0, i: 1, d: 10.0}
